src/main/java/com/denizenscript/depenizen/bukkit/commands/mythicmobs/MythicSkillCommand.java

```diff
@@ -0,0 +1,111 @@
+package com.denizenscript.depenizen.bukkit.commands.mythicmobs;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizen.utilities.Utilities;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
+import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.ArgumentHelper;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import com.denizenscript.depenizen.bukkit.bridges.MythicMobsBridge;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MythicSkillCommand extends AbstractCommand {
+
+    public MythicSkillCommand() {
+        setName("mythicskill");
+        setSyntax("mythicskill [<skillname>] (<location>|...) (<entity>|...) (power:<#.#>) (casters:<entity>|..)");
+        setRequiredArguments(1, 5);
+    }
+
+    // <--[command]
+    // @Name MythicSkill
+    // @Syntax mythicskill [<skillname>] (<location>|...) (<entity>|...) (power:<#.#>) (casters:<entity>|..)
```

Comment:
If `You may not have both the Entity and Location targets, one or the other.` is true, the meta should probably say something more like `(<location>|.../<entity>|...)`